src/components/Patient/PatientDetailsTab/PatientUsers.tsx

```diff
@@ -191,7 +191,7 @@ function AddUserSheet({ patientId }: AddUserSheetProps) {
                   <SelectTrigger data-cy="patient-user-role-select">
                     <SelectValue placeholder={t("select_role")} />
                   </SelectTrigger>
-                  <SelectContent>
+                  <SelectContent className="min-w-[var(--radix-select-trigger-width)] max-w-[90vw]">
```

Comment:
we can simply do this right? 🤔 correct me if i am wrong

```suggestion
                  <SelectContent className="w-[var(--radix-select-trigger-width)]">
```

Reply:
We have to set the min width otherwise it take width according to the content. Although we can remove max width.

Reply:
Oh yes we can do this. It is unecessary to set min and max width😅